### DISCUSSION TITLE
Howto change admin password?

### DISCUSSION THREAD

**Discussion by tekstacey** [mealie-recipes/mealie#1482] [Q&A]
I have forgotten the admin user password.
I have tried the default changeme account but that doesn't exist anymore.

How can I reset the admin password, or add a new admin user?

Or is there a way top set an admin login during build?

**Comment by michael-genson** [comment]
Please see the FAQ for this here: https://docs.mealie.io/documentation/getting-started/faq/#how-can-i-change-my-password
